plasTeX/Imagers/__init__.py

```diff
@@ -518,12 +518,14 @@ def writePreamble(self, document):
 
     def verify(self):
         """ Verify that this commmand works on this machine """
-        if self.verification:
-            proc = os.popen(self.verification)
-            proc.read()
-            if not proc.close():
-                return True
-            return False
+        if self.verifications:
```

Comment:
This is not directly related to this PR, but since you changed something here, maybe it would be a good opportunity to include more feed-back when this fails. For instance, instead of returning `False` you could return the verification that failed, and then in [`Renderer.__init__`](https://github.com/plastex/plastex/blob/acaa91fbaee4bd826a941d3a8fce18e9a4a18415/plasTeX/Renderers/__init__.py#L450-L452) change the test and write what failed when it fails. If you choose to do that then don't forget to update [the docs](https://github.com/plastex/plastex/blob/8ebba0fa05666da8b0928ea2a40f389f6a64e454/Doc/imager-api.tex#L179-L183).

Reply:
I suppose one way to do this is for `verify` to raise an exception when it is missing something, and the `Renderer` would catch this error and print the corresponding message.

Reply:
Yes, let's keep that for a future PR.